### ISSUE TITLE
change docker images

### ISSUE THREAD

**Issue by xiphl** [neojunjie/datahub#1]
**Is your feature request related to a problem? Please describe.**
Docker will encounter errors running `RUN apk --no-cache add curl tar     && curl -L https://github.com/jwilder/dockerize/releases/download/v0.6.1/dockerize-linux-amd64-v0.6.1.tar.gz | tar -C /usr/local/bin -xzv` in networks behind proxy. 
`#5 1.166 curl failed to verify the legitimacy of the server and therefore could not
#5 1.166 establish a secure connection to it.`
basically any github/releases seemed to always have the problem.

**Describe the solution you'd like**
change image to one that already has the existing file downloaded.
